src/main.ts

```diff
@@ -2,8 +2,46 @@
 
 import * as ts from 'typescript';
 
-class Transpiler {
+const DEBUG = true;
+
+export const options: ts.CompilerOptions = {
+  allowNonTsExtensions: true,
+  module: ts.ModuleKind.CommonJS,
+  target: ts.ScriptTarget.ES5,
+};
+
+export class Minifier {
   constructor() {}
 
-  helloWorld() { return 'hello world'; }
-}
\ No newline at end of file
+  visit(node: ts.Node) {
+    switch (node.kind) {
+      case ts.SyntaxKind.PropertyAccessExpression:
+        result = '';
+        var pae = <ts.PropertyAccessExpression>node;
+        result += this.visit(pae.expression);
+        result += '.';
+        // Adds '$mangled' to an identifier to ensure that the identifier is being altered.
+        result += pae.name.text + '$mangled';
+        break;
+
+      default:
+        var text = node.getText();
+        var children = node.getChildren();
```

Comment:
consider wrapping this block in a `{ }` block and using `let` for the variables, then you don't get cross talk between the different `case` statement variables.